src/ui/Quickview/Quickview.ts

```diff
@@ -42,7 +42,9 @@ interface IQuickviewOpenerObject {
  * information about a result. Most of the time, this component references a
  * [`QuickviewDocument`]{@link QuickviewDocument} in its [`contentTemplate`]{@link Quickview.options.contentTemplate}.
  *
- * **Note:**
+ * **Notes:**
+ * > - `Quickview` is not a replacement for simply using the result link to consult the document,
```

Comment:
Suggestion:

A `Quickview` is not meant to replace a [`ResultLink`]{@link ResultLink} to access an item; it has certain limitations (e.g., embedded images and links may not work as expected in a `Quickview`).